pkg/network/infraconfigurators/bridge.go

```diff
@@ -46,6 +46,9 @@ func (b *BridgePodNetworkConfigurator) DiscoverPodNetworkInterface(podIfaceName
 		log.Log.Reason(err).Errorf("failed to get a link for interface: %s", podIfaceName)
```

Comment:
This is a really good question. All the times I've seen this code I wondered about why it is needed, and couldn't come up w/ a reason for it.

Maybe we're better off just removing that validation, since, it does appear to be pointless.

I'll push a PR removing that asap. We'll move this discussion there if you don't mind.

Reply:
please review the dependent PR mentioned above @qinqon / @alonSadan

Reply:
Thanks. I will